checkout/domain/placeorder/process/process.go

```diff
@@ -166,6 +187,9 @@ func (p *Process) Run(ctx context.Context) {
 		return
 	}
 
+	censusCtx, _ := tag.New(ctx, tag.Upsert(opencensus.KeyArea, "-"), tag.Upsert(keyState, currentState.Name()))
```

Comment:
Why overriding the area here? I guess we could just remove that.

Reply:
Wasn't sure what to do with this "area". If it's fine that it's empty, that's okay for me as well.

Reply:
jeah just remove the whole `tag.Upsert(opencensus.KeyArea, "-"),` since it isn't any value.

Reply:
Injected the area config now as discussed and added the area to the metrics.